stripper/ze_skill_escape_v09.cfg

```diff
@@ -47,39 +47,6 @@ modify:
 	}
 }
 
-;If zombie is at top when zms tp to cage, end game.
-add:
-{
-	"wait" "1"
-	"targetname" "ZMWatcher"
-	"StartDisabled" "1"
-	"spawnflags" "1"
-	"origin" "3907.5 3735.5 -311.5"
-	"classname" "trigger_multiple"
-	"filtername" "filterzombies"
-	"OnStartTouch" "server,Command,say << ZOMBIE REACHED THE TOP >>,0,1"
-	"OnStartTouch" "server,Command,say << ZOMBIE REACHED THE TOP >>,1,1"
-	"OnStartTouch" "server,Command,say << ZOMBIE REACHED THE TOP >>,2,1"
-	"OnStartTouch" "server,Command,say << GAME OVER >>,3,1"
-	"OnStartTouch" "player,AddOutput,origin 2124 3736 66,4,1"
-	"OnStartTouch" "trigger_teleport,Enable,,5,1"
-}
-
-modify:
-{
-	match:
-	{
-		"classname" "logic_auto"
-	}
-	insert:
-	{
-		"OnMapSpawn" "ZMWatcher,AddOutput,solid 2,0,1"
-		"OnMapSpawn" "ZMWatcher,AddOutput,mins -537.5 -544.5 -258.5,0.05,1"
-		"OnMapSpawn" "ZMWatcher,AddOutput,maxs 537.5 544.5 258.5,0.05,1"
-		"OnMapSpawn" "ZMWatcher,AddOutput,targetname lasersztp,0.1,1"
-	}
-}
-
```

Comment:
If possible we should really try to fix and keep this change. This map and the mapper do come from Mapeadores, so as far as the stripper goes it can be considered an official intended change.

I took a quick look in Hammer for the low gravity trap you mentioned, but I'm not super familiar with the map and didn't find anything.

Reply:
Readded the fix as is. Low grav trap is the one with the melon next to the caution square